ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -706,9 +706,10 @@ suggestModuleTypo Diagnostic{_range=_range,..}
 --     Could not find module ‘Data.Cha’
 --     Perhaps you meant Data.Char (from base-4.12.0.0)
     | "Could not find module" `T.isInfixOf` _message
-    , "Perhaps you meant"     `T.isInfixOf` _message = let
-      findSuggestedModules = map (head . T.words) . drop 2 . T.lines
-      proposeModule mod = ("replace with " <> mod, TextEdit _range mod)
+    , "Perhaps you meant"     `T.isInfixOf` _message =
+      -- TODO findSuggestedModules never extracts module name - fix the parsing logic
```

Comment:
This parsing logic is definitely broken. If I'm not wrong, this will always extract the word "Perhaps" ?!

Reply:
Did you check the tests?

Reply:
This doesn't seem to be tested at all.
Also no action is generated for these kinds of errors, probably because the "Perhaps you meant" in on the 2nd line of the message, so it gets dropped by the drop 2 part of the original impl.
Let me add test and fix this in separate PR.